pmg/static/resources/javascript/committees.js

```diff
@@ -54,70 +59,50 @@ var debounce = function(func, wait, immediate) {
 	};
 };
 
-var typeAhead = debounce(function($input,$list,$res,params) {
-  var value = $input.val();
+var renderSearchResults = function(results,$list,$res,params) {
+  var resultsCount = results.length;
   var $noResults = $res.find('.no-results');
   var $resultsList = $res.find('.results-list');
-  var $items = !!params && params.$items && params.$items.length ? params.$items : $list.find('.item');
-
-  clearSearchResult($list,$res,!!params && params.twoCol ? params.twoCol : false);
-
-  if(!!value.length) {
-    var $results = [];
-    var resultsCount = 0;
-
-    showingSearchResult = true;
 
-    $list.hide();
-    $noResults.hide();
+  //clearSearchResult($list,$res,params);
 
-    $items.each(function() {
-      var $self = $(this);
+  showingSearchResult = true;
 
-      if($self.find('a').text().toLowerCase().includes(value.toLowerCase())) {
-        $results.push($self.clone());
-      }
-    });
+  $list.hide();
+  $noResults.hide();
 
-    resultsCount = $results.length;
+  if(!!resultsCount) {
+    $resultsList.show();
 
-    if(!!resultsCount) {
-      $resultsList.show();
+    if(!!params && params.sort) {
+      results.sort(function(a,b) {
+        return new Date($(b).find('.date').text()) - new Date($(a).find('.date').text());
```

Comment:
If the date strings are 'yyyy-mm-dd', then just sorting the text strings is the same as converting to dates and sorting. So you don't strictly need to convert to dates.